addons/hr_attendance/static/src/js/kiosk_mode.js

```diff
@@ -55,9 +56,20 @@ var KioskMode = AbstractAction.extend({
 
     destroy: function () {
         core.bus.off('barcode_scanned', this, this._onBarcodeScanned);
+        clearInterval(this._interval);
         clearInterval(this.clock_start);
         this._super.apply(this, arguments);
     },
+
+    call2Db: function () { 
+        return this._rpc({
+               model: 'res.users',
+               method: 'search_read',
+           })
+           .then(function (users){
+               console.log(users[0].id);
```

Comment:
@LofbergValentin Is this a remaining debugging chunk of code ?

Reply:
You are right fixing that right away